### PR TITLE
Fixed console example

### PR DESCRIPTION
**Pull request by iamonzon**
Added Monad constraint and removed extra implicit

### CODE REVIEW

File: std/js/src/main/scala/cats/effect/std/Console.scala [hunk 1]
```diff
@@ -32,18 +32,16 @@ import java.nio.charset.Charset
...
  *
- * implicit val console = Console.sync[F]
- *
- * def myProgram[F[_]: Console]: F[Unit] =
- *   for {
- *     _ <- Console[F].println("Please enter your name: ")
- *     n <- Console[F].readLine
- *     _ <- if (n.nonEmpty) Console[F].println("Hello, " + n) else Console[F].errorln("Name is empty!")
- *   } yield ()
+ *  def myProgram[F[_] : Console : Monad]: F[Unit] =
```
Comment (armanbilge) [inline std/js/src/main/scala/cats/effect/std/Console.scala]: I _think_ this is the style enforced by our scalafmt, but I personally don't care :)

```suggestion
 *  def myProgram[F[_]: Console: Monad]: F[Unit] =
```

**Review by armanbilge — DISMISSED** [review]
Thanks for fixing this, LGTM! Much appreciated :)

Ideally we'd set up something like [sbt-doctest](https://github.com/tkawachi/sbt-doctest) to avoid this in the future :)

File: std/js/src/main/scala/cats/effect/std/Console.scala (hunk 1 above)
Comment (iamonzon) [reply std/js/src/main/scala/cats/effect/std/Console.scala]: Oh, you're right. 
I thought I've ran `sbt prePR` to make sure it was in the right format. My bad, I accepted the commit

PS: I didn't realize it was going to require approbation again :sweat_smile:

Comment (armanbilge) [reply std/js/src/main/scala/cats/effect/std/Console.scala]: I don't think scalafmt can check/format code in scaladocs, so it's not your fault :) thanks!

**Review by armanbilge — APPROVED** [review]

File: std/js/src/main/scala/cats/effect/std/Console.scala [hunk 2]
```diff
@@ -32,18 +32,16 @@ import java.nio.charset.Charset
  *
  * @example
  *   {{{
- * import cats.effect.std.Console
- * import cats.effect.kernel.Sync
- * import cats.syntax.all._
+ *  import cats.Monad
+ *  import cats.effect.std.Console
+ *  import cats.syntax.all._
```
Comment (samirbajaj) [inline std/js/src/main/scala/cats/effect/std/Console.scala:37]: Nitpick: why the leading space?

Comment (djspiewak) [reply std/js/src/main/scala/cats/effect/std/Console.scala:37]: I would guess scalafmt did something odd. It seems to really struggle with code snippets in scaladoc.

**Pull request merged**